Zubkova_A_task02

sandriiia opened the pull request:
Я создала в своей папке task02 папку level1(как в примере) и тесты были для нее

sandriiia committed:
task02

Я создала в своей папке task02 папку level1(как в примере) и тесты были для нее

xthebat commented on Zubkova_A/task02/main.py:
```diff
@@ -0,0 +1,21 @@
+
+def file(path, level=1):
+    print('level=', level, 'Content:', os.listdir(path))
+    for i in os.listdir(path):
```
Некорректный нейминг переменных

xthebat commented on Zubkova_A/task02/main.py:
```diff
@@ -0,0 +1,21 @@
+import os
+
+
+def file(path, level=1):
```
Не соблюдение ТЗ, функция сбора данных и вывода должны быть разделены

xthebat commented on Zubkova_A/task02/main.py:
```diff
@@ -0,0 +1,21 @@
+    print('level=', level, 'Content:', os.listdir(path))
+    for i in os.listdir(path):
+        if os.path.isdir(path+'\\'+i):
+            print('entrance', path+'\\'+i)
```
Предпочтительно использовать `os.path.join` для объединения пути

xthebat reviewed (commented):
Score: 1/4

sandriiia committed:
Исправления

Исправила нейминг переменных, поменяла объеденение пути.
Функцию для вывода данных вынести не получилось и не успела до дедлайна, из за проблем с рекурсией

xthebat commented:
Score: 2/4 (необходимо было разделить на две функции, читайте задание внимательней)
